packages/owl-bot/src/owl-bot.ts

```diff
@@ -59,6 +59,7 @@ import {shouldIgnoreRepo} from './should-ignore-repo';
 // We use lower case organization names here, so we need to always
 // check against lower cased owner.
 const ALLOWED_ORGANIZATIONS = ['googleapis', 'googlecloudplatform'];
+const ALLOWED_FORK_OWNERS = ['yoshi-code-bot'];
```

Comment:
super minor nit but also acknowledging the prior usage of a list: we could use a `Set` here.

Reply:
For a small list like this, it probably doesn't matter in terms of performance.